lib/lightspeed/item.rb

```diff
@@ -58,5 +58,10 @@ class Item < Lightspeed::Resource
     def prices
       @prices ||= Lightspeed::Prices.new(self.Prices)
     end
+
+    def default_vendor
```

Comment:
oh ok, cool. It will derive the `defaultVendorID` key automatically? Is this at all confusing since the item should also have another relationship to `Vendor` via `ItemVendorNums`?